src/main/scala/org/apache/spark/sql/delta/commands/WriteIntoDelta.scala

```diff
@@ -122,6 +122,13 @@ case class WriteIntoDelta(
       case _ => Nil
     }
 
-    newFiles ++ deletedFiles
+    val rearrangeOnly = options.rearrangeOnly
+    if (rearrangeOnly) {
+      newFiles.map(_.copy(dataChange = !rearrangeOnly)) ++
+        deletedFiles.map(_.copy(dataChange = !rearrangeOnly))
+    }
+    else {
```

Comment:
nit: incorrect indent.

Reply:
Sorry, I can't see the wrong ident :). I thought that when we need to wrap an expression across more than one line, each successive line should be indented two spaces from the first.
isn't that correct ?

Reply:
Should be:
`} else {`
on one line. 

Ideally the linter would catch this sort of thing... I'll do some looking into if we're missing a scalastyle rule.